### PR TITLE
22 hamcrest in tests

### LINKED ISSUE #22: eo-hamcrest usage
Let's get start to use `eo-hamcrest` everywhere in this project tests.

### PR DESCRIPTION
**Pull request by includealex**
Closes #22

### CODE REVIEW

**Comment by includealex** [comment]
@Graur please take a look.

**Comment by Graur** [comment]
@includealex Can you change this version please: https://github.com/objectionary/eo-files/blob/master/pom.xml#L107

**Comment by includealex** [comment]
@Graur `3.10.1` is the newest version now. Should I change it back to `3.8.1` to restore changes of [this](https://github.com/objectionary/eo-files/pull/27) pull request?

**Comment by Graur** [comment]
@includealex Yep

**Comment by includealex** [comment]
@Graur please have a look

File: src/test/eo/org/eolang/fs/file-test.eo [hunk 1]
```diff
@@ -26,87 +26,114 @@
...
+    eq.
+      copied
+        bytes-as-input
+          as-bytes.
+            line
+        f.as-output "w+"
+        100
+      f.size
+    $.equal-to
+      TRUE
```
Comment (Graur) [inline src/test/eo/org/eolang/fs/file-test.eo:139]: @includealex Why don't you change this `eq` object to `assert-that` object?

**Review by Graur — CHANGES_REQUESTED** [review]
@includealex Good one, just one comment from my side:

File: src/test/eo/org/eolang/fs/file-test.eo (hunk 1 above)
Comment (includealex) [reply src/test/eo/org/eolang/fs/file-test.eo:139]: @Graur if we try to change `eq` object to `equal-to`:
```eo
[] > measures-file-size
  tmpdir.tmpfile > f!
  "hey!" > line
  assert-that > @
    copied
      bytes-as-input
        as-bytes.
          line
      f.as-output "w+"
      100
    $.equal-to
      f.size

```
we will find out that `f.size` is equal to `0`.

Comment (Graur) [reply src/test/eo/org/eolang/fs/file-test.eo:139]: @includealex So, this test fail?

Comment (includealex) [reply src/test/eo/org/eolang/fs/file-test.eo:139]: @Graur yes, it looks like a bug.

**Comment by includealex** [comment]
@Graur please check this one

**Review by Graur — APPROVED** [review]
@yegor256 please, have a look

**Pull request merged**